wpimath/src/main/java/edu/wpi/first/math/trajectory/struct/ExponentialProfileStateStruct.java

```diff
@@ -0,0 +1,39 @@
+package edu.wpi.first.math.trajectory.struct;
+
+import edu.wpi.first.math.trajectory.ExponentialProfile;
+import edu.wpi.first.util.struct.Struct;
+
+import java.nio.ByteBuffer;
+
+public class ExponentialProfileStateStruct implements Struct<ExponentialProfile.State> {
+	@Override
+	public Class<ExponentialProfile.State> getTypeClass() {
+		return ExponentialProfile.State.class;
+	}
+	
+	@Override
+	public String getTypeName() {
+		return "TrapezoidProfile.State";
```

Comment:
```suggestion
		return "ExponentialProfileState";
```
The type name can't contain `.`. Note that `Trajectory.State` was named `TrajectoryState` in https://github.com/wpilibsuite/allwpilib/blob/main/wpimath/src/main/proto/trajectory.proto>.